Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM node:12.14
```

Comment:
I wonder if we can use node-alpine?

Reply:
I'm sure we could use Alpine, I'll have a go at converting it, and submit another PR